test/TaraClient.t.sol

```diff
@@ -82,6 +82,18 @@ contract TaraClientTest is Test {
         return CompactSignature(r, (bytes32(uint256(v)) << 255) | s);
     }
 
+    function sortSignatures(CompactSignature[] memory signatures) public pure returns (CompactSignature[] memory) {
```

Comment:
Hmm maybe it is a good idea to put this method as a pure one on the contract itself that takes the signatures? Maybe someone will use it.

Reply:
I don't see why anyone will need this. It is not hard to sort it in the script